CONTRIBUTING.md

```
@@ -226,34 +226,50 @@ To maintain a clean and linear project history, we prefer rebasing your feature
 
+
 3.  **Resolve conflicts**: If there are merge conflicts, Git will pause the rebase and ask you to resolve them. After resolving conflicts, continue the rebase:
 
     ```bash
     git add . # Add resolved files
     git rebase --continue
     ```
 
+
+    > **💡 GUI Alternative**: VSCode will show conflict markers in your files. Resolve conflicts using the inline conflict resolution tools, then use the Source Control panel to stage changes and continue the rebase.
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)